src/components/AttachmentModal.js

```diff
@@ -257,9 +267,15 @@ class AttachmentModal extends PureComponent {
                         onDownloadButtonPress={() => this.downloadAttachment(source)}
                         onCloseButtonPress={() => this.setState({isModalOpen: false})}
                     />
-
                     <View style={styles.imageModalImageCenterContainer}>
-                        {this.state.source && this.state.shouldLoadAttachment && (
+                        {this.state.reportID ? (
```

Comment:
https://github.com/Expensify/App/issues/15809 regression was caused by this PR. When user navigates to the details of a user through participants list of a group chat, the route contains the reportID and hence user avatar is displayed with `<AttachmentCarousel>` instead of `<AttachmentView>`.